java/0280-wiggle-sort.java

```diff
@@ -0,0 +1,14 @@
+class Solution {
+    public void wiggleSort(int[] nums) {
+        for(int i = 1; i < nums.length; i++){
+            if((i % 2 == 0 && nums[i] > nums[i - 1]) 
+            || (i % 2 == 1 && nums[i] < nums[i - 1]))
```

Comment:
It's better to have a space after control flow statements. There should also be an additional space between the for loop and the opening bracket

Reply:
Added spacing before the control flow statements